src/libraries/System.Private.CoreLib/src/System/Int128.cs

```diff
@@ -1904,7 +1904,8 @@ static bool INumberBase<Int128>.TryConvertToSaturating<TOther>(Int128 value, [Ma
             }
             else if (typeof(TOther) == typeof(ulong))
             {
-                ulong actualResult = (value <= 0) ? ulong.MinValue : (ulong)value;
+                ulong actualResult = (value <= 0) ? ulong.MinValue :
+                                     (value >= ulong.MaxValue) ? ulong.MaxValue : (ulong)value;
```

Comment:
Can we make this exactly match the structure of the ushort and uint cases above it? e.g.
```suggestion
                ulong actualResult = (value >= ulong.MaxValue) ? ulong.MaxValue :
                                     (value <= ulong.MinValue) ? ulong.MinValue : (ulong)value;
```